images/apko/image.yaml

```diff
@@ -1,5 +1,6 @@
 versions:
   - melange:
-      config: configs/latest.melange.yaml
+      configs:
+        - configs/latest.melange.yaml
```

Comment:
Just a quick question — if we're deciding to support both `config` and `configs` at the same time, why are we changing these instances where there's just a single config to use the multi-config option?

Reply:
the `multi-config` option supports a comma-separated list. In the case of a single melange file, this is still a valid list (with just a single item)

We could leave this as `config: configs/latest.melange.yaml`, but I thought it'd be easier for people copy-pasting later who need multiple melange configs to be consistent everywhere

Reply:
nevermind actually.. I just dropped support for the single config altogether since it isnt being used anywhere

Reply:
Nice!